src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -83,208 +83,253 @@ class ReviewCollapsibleChapter extends React.Component {
     });
   };
 
-  render() {
-    let pageContent = null;
+  getChapterTitle = chapterFormConfig => {
+    let chapterTitle = chapterFormConfig.title;
+    if (typeof chapterFormConfig.title === 'function') {
+      chapterTitle = chapterFormConfig.title(true);
+    }
+    if (chapterFormConfig.reviewTitle) {
+      chapterTitle = chapterFormConfig.reviewTitle;
+    }
+    return chapterTitle;
+  };
 
+  getSchemaformPageContent = (page, props) => {
```

Comment:
This is refactored from what was in `render()` with a few changes called out in the review comments.